### PR TITLE
Consume events for websocket connections instead of polling

### PR DESCRIPTION
**Pull request by AIIX**
- Websocket connector interface now uses the internal home assistant event system for updating device models
- Removes / Disables the polling mechanism from websocket api
- Fixes get device api calls to provide the correct display models
- Fixes MediaPlayerCard layout which was causing layout polish issues

### CODE REVIEW

File: ovos_PHAL_plugin_homeassistant/logic/connector.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 from ovos_utils.log import LOG
 from ovos_PHAL_plugin_homeassistant.logic.socketclient import HomeAssistantClient
+import nested_lookup
```
Comment (NeonDaniel) [inline ovos_PHAL_plugin_homeassistant/logic/connector.py:10]: Looks like a new dependency missing from `requirements.txt`

```
2023-02-02 14:28:47.981 - enclosure - ovos_plugin_manager.utils:find_plugins:89 - ERROR - Failed to load plugin entry point ovos-PHAL-plugin-homeassistant = ovos_PHAL_plugin_homeassistant:HomeAssistantPlugin
Traceback (most recent call last):
  File "/home/neon/venv/lib/python3.7/site-packages/ovos_plugin_manager/utils/__init__.py", line 87, in find_plugins
    entrypoints[entry_point.name] = entry_point.load()
  File "/home/neon/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2472, in load
    return self.resolve()
  File "/home/neon/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2478, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/neon/venv/lib/python3.7/site-packages/ovos_PHAL_plugin_homeassistant/__init__.py", line 7, in <module>
    from ovos_PHAL_plugin_homeassistant.logic.connector import HomeAssistantRESTConnector, HomeAssistantWSConnector
  File "/home/neon/venv/lib/python3.7/site-packages/ovos_PHAL_plugin_homeassistant/logic/connector.py", line 10, in <module>
    import nested_lookup
ModuleNotFoundError: No module named 'nested_lookup'
```

**Review by NeonDaniel — CHANGES_REQUESTED** [review]
`nested_lookup 0.2.25` installed and plugin loads after that for me
Also has ovos-phal-plugin-oauth dependency that isn't listed

Otherwise, looks good and working for me with UI updates when connected via `wss`

**Comment by NeonDaniel** [comment]
Rebased on `dev`. Should be fine with rebase (up to you if you want to rebase locally or with the web editor)

**Comment by AIIX** [comment]
should be fixed now

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**